<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSAR">
    <property name="title">
     <string>SAR基本工具</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_bianhua">
    <property name="title">
     <string>变化检测处理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_houchuli">
    <property name="title">
     <string>后处理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menuSAR"/>
   <addaction name="menu_bianhua"/>
   <addaction name="menu_houchuli"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
